integrated_channels/moodle/client.py

```diff
@@ -150,6 +151,100 @@ def _get_access_token(self):
                 response.status_code
             )
 
+    @moodle_request_wrapper
+    def _get_enrolled_users(self, course_id):
+        """
+        placeholder
+        """
+        params = {
+            'wstoken': self.token,
+            'wsfunction': 'core_enrol_get_enrolled_users',
```

Comment:
these jokers really spell it `enrol` and `enrolment`, but then use `enrolled`. _flips table_

Reply:
...As I'm sure you know already, that's not even the worst part! They have function names misspelled in their docs that don't have a list of parameters so when you grep through the source code nothing comes up 😞

Reply:
oh yes like grades vs grade

Reply:
😠  agreed